test_scripts/Capabilities/PersistingHMICapabilities/006_HMICapabilityCacheFile_does_not_exist_in_AppStorage_folder.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0249-Persisting-HMI-Capabilities-specific-to-headunit.md
+--
+-- Description: Check that SDL sends correspondent HMI capabilities request (VR/TTS/RC/UI/Buttons/VehicleInfo etc)
+--  in case one of them is missing in HMI capabilities cache ("hmi_capabilities_cache.json") file
+--
+-- Preconditions:
+-- 1. Value of HMICapabilitiesCacheFile parameter is defined (hmi_capabilities_cache.json) in smartDeviceLink.ini file
+-- 2. hmi_capabilities_cache.json doesn't exist on file system
+-- 3. SDL and HMI are started
+-- 4. HMI does not provide one of HMI capabilities (VR/TTS/RC/UI/Buttons/VehicleInfo etc)
+-- Sequence:
+-- 1. IGN_OFF/IGN_ON
+--  a. SDL sends correspondent HMI capabilities (VR/TTS/RC/UI/Buttons/VehicleInfo etc) request to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Capabilities/PersistingHMICapabilities/common')
+
+--[[ Local Variable ]]
+local hmiDefaultCap = common.getDefaultHMITable()
+
+local cap = {
+  UI = { "GetLanguage", "GetSupportedLanguages", "GetCapabilities" },
+  VR = { "GetLanguage", "GetSupportedLanguages", "GetCapabilities" },
+  TTS = { "GetLanguage", "GetSupportedLanguages", "GetCapabilities" },
+  Buttons = { "GetCapabilities" },
+  VehicleInfo = { "GetVehicleType" },
+  RC = { "GetCapabilities" }
+}
+
+--[[ Local Functions ]]
+local function getHMIParamsWithOutResponse(pMod, pReq)
+  local noResponseGetHMIParams = common.cloneTable(hmiDefaultCap)
+  noResponseGetHMIParams[pMod][pReq] = nil
+  return noResponseGetHMIParams
+end
+
+local function getHMIParamsWithOutRequest(pMod, pReq)
+  local requestGetHMIParams = common.getHMIParamsWithOutRequests()
+  requestGetHMIParams[pMod][pReq].occurrence = nil
```

Comment:
Shouldn't this line set `occurrence = 1` since what we really just want is that one request to be resent since it did not receive response last time?